backend/models/institution.py

```diff
@@ -319,15 +319,18 @@ def is_active(self):
         """Check if is active."""
         return self.state == "active"
     
-    def get_all_hierarchy_admin_permissions(self, permissions=None):
+    def get_hierarchy_admin_permissions(self, get_all=True, admin_key=None, permissions=None):
         """
         This method get all hierarchy admin permissions of
         institution. When result, returns a dict containing
         the admin permissions of all institutions belonging 
         to the child hierarchy.
 
         Arguments:
-        permissions(Optional) -- Dict of previous permissions added for add more permissons. 
```

Comment:
I agree with @Luiz-FS there is some semantic error. get_all = True gotta mean: The method will get all the permissions.